Python/PRP/GUI/pyGUIButtonMod.cpp

```diff
@@ -0,0 +1,188 @@
+        PyTuple_SET_ITEM(list, i, pyPlasma_convert(self->fThis->getAnimationKeys()[i]));
+    return list;
+}
+PY_PROPERTY_SETTER_MSG(GUIButtonMod, animationKeys, "To add animations, use addAnim()")
+PY_PROPERTY_GETSET_DECL(GUIButtonMod, animationKeys)
+
+PY_GETSET_GETTER_DECL(GUIButtonMod, mouseOverKeys) {
+    PyObject* list = PyTuple_New(self->fThis->getMouseOverKeys().size());
+    for (size_t i = 0; i<self->fThis->getMouseOverKeys().size(); i++)
+        PyTuple_SET_ITEM(list, i, pyPlasma_convert(self->fThis->getMouseOverKeys()[i]));
+    return list;
+}
+PY_PROPERTY_SETTER_MSG(GUIButtonMod, mouseOverKeys, "To add mouse-over animations, use addMouseOver()")
+PY_PROPERTY_GETSET_DECL(GUIButtonMod, mouseOverKeys)
+
+PY_PROPERTY(ST::string, GUIButtonMod, animName, getAnimationName, setAnimationName)
+PY_PROPERTY(ST::string, GUIButtonMod, mouseOverAnimName, getMouseOverAnimName, setMouseOverAnimName)
+PY_PROPERTY(plKey, GUIButtonMod, draggable, getDraggable, setDraggable)
+PY_PROPERTY(int, GUIButtonMod, notifyType, getNotifyType, setNotifyType)
+
+static PyGetSetDef pyGUIButtonMod_GetSet[] = {
+    pyGUIButtonMod_animName_getset,
+    pyGUIButtonMod_mouseOverAnimName_getset,
+    pyGUIButtonMod_draggable_getset,
+    pyGUIButtonMod_notifyType_getset,
+    pyGUIButtonMod_animationKeys_getset,
+    pyGUIButtonMod_mouseOverKeys_getset,
+    PY_GETSET_TERMINATOR
+};
+
+PY_PLASMA_TYPE(GUIButtonMod, pfGUIButtonMod, "pfGUIButtonMod wrapper");
+
+PY_PLASMA_TYPE_INIT(GUIButtonMod) {
+    pyGUIButtonMod_Type.tp_new = pyGUIButtonMod_new;
+    pyGUIButtonMod_Type.tp_methods = pyGUIButtonMod_Methods;
+    pyGUIButtonMod_Type.tp_getset = pyGUIButtonMod_GetSet;
+    pyGUIButtonMod_Type.tp_base = &pyGUIControlMod_Type;
+    if (PyType_Ready(&pyGUIButtonMod_Type) < 0)
+        return NULL;
+
+    /* Konstants */
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnUp", pfGUIButtonMod::kNotifyOnUp);
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnDown", pfGUIButtonMod::kNotifyOnDown);
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnUpAndDown", pfGUIButtonMod::kNotifyOnUpAndDown);
+
+    Py_INCREF(&pyGUIButtonMod_Type);
+    return (PyObject*)&pyGUIButtonMod_Type;
+}
+
+PY_PLASMA_IFC_METHODS(GUIButtonMod, pfGUIButtonMod)
+
+}
+
+extern "C" {
+
+PY_PLASMA_NEW(GUIMenuItem, pfGUIMenuItem)
+
+PY_PLASMA_TYPE(GUIMenuItem, pfGUIMenuItem, "pfGUIMenuItem wrapper");
+
+PY_PLASMA_TYPE_INIT(GUIMenuItem) {
+    pyGUIMenuItem_Type.tp_new = pyGUIMenuItem_new;
+    pyGUIMenuItem_Type.tp_base = &pyGUIButtonMod_Type;
+    if (PyType_Ready(&pyGUIMenuItem_Type) < 0)
+        return NULL;
+
+    /* Konstants */
+    PY_TYPE_ADD_CONST(GUIMenuItem, "kDrawSubMenuArrow", pfGUIMenuItem::kDrawSubMenuArrow);
+    PY_TYPE_ADD_CONST(GUIMenuItem, "kReportHovers", pfGUIMenuItem::kReportHovers);
+
+    Py_INCREF(&pyGUIMenuItem_Type);
+    return (PyObject*)&pyGUIMenuItem_Type;
+}
+
+PY_PLASMA_IFC_METHODS(GUIMenuItem, pfGUIMenuItem)
+
+}
```

Comment:
Bah, these don't go away when I fix them because this line isn't affected by adding a new one below it. 🤕